### PR TITLE
add question file

### PR DESCRIPTION
**Pull request by eliyahubowers**
the questions

### CODE REVIEW

**Comment by KevinWh0** [comment]
I think that these questions need some work:
burn down 1/2 of the forest - Why would you burn down 1/2 of the forest? You want to make paper, so why would burning it down help you in any way??
hire protestors - I doubt that the protesters would want to be hired by a company that they are protesting against
licesnce removal of forest - I don't understand this (does it mean, get a license to chop the forest, or sell a license?)
a hospital needs space for ading the sick - I don't understand this one

The other questions make sense, it's just these ones that don't so you might want to review those.

**Review by ChaoticLeah — CHANGES_REQUESTED** [review]
I think that these questions need some work:
burn down 1/2 of the forest - Why would you burn down 1/2 of the forest? You want to make paper, so why would burning it down help you in any way??
hire protestors - I doubt that the protesters would want to be hired by a company that they are protesting against
licesnce removal of forest - I don't understand this (does it mean, get a license to chop the forest, or sell a license?)
a hospital needs space for ading the sick - I don't understand this one

The other questions make sense, it's just these ones that don't so you might want to review those.

**Comment by KevinWh0** [comment]
I think that these questions need some work:
burn down 1/2 of the forest - Why would you burn down 1/2 of the forest? You want to make paper, so why would burning it down help you in any way??
hire protestors - I doubt that the protesters would want to be hired by a company that they are protesting against
licesnce removal of forest - I don't understand this (does it mean, get a license to chop the forest, or sell a license?)
a hospital needs space for ading the sick - I don't understand this one

The other questions make sense, it's just these ones that don't so you might want to review those.

File: questions.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{ burn down 1/2 of the forest ; yes : it gets burnt down; no : it doesnt get burnt down}
+{ hire more staff ; hire 1 : you hire a worker; hire 2 : you hire 2 workers; hire 3 : you hire three workers}
+{ hire protestors ; no : nothing changes; yes : small increase of awareness; alot : large increase of awareness}
+{ licesnce removal of forest ; yes : gain  $1,000-$10,000; no : public approval increases ; alot : gain $10,000-$100,000}
+{ invest in other companies? ; no : nothing ; a little : small immedieate decrease in $ later posibility of some$ ; more : middle ; most : large immedieate decrease in $ later posibility of larger $}
+{ a hospital needs space for ading the sick; yes : public approval increases ; no : public approval idecreases}
+{ your forest is burning, give money for forest fighting; none : it burns ; $10,000 : decreases somewhaT ; $100,000 : decreases significantly}
```
Comment (kronodeus) [inline questions.txt]: I really like this one! Makes you choose between two resources which is a good dynamic for the player

**Review by ChaoticLeah — COMMENTED** [review]
Might want to describe what Coverup means. Also does license removal of forest mean cut it down or what? Also by increase the quality of living for natives, do you mean the people who live in the forest(Indians, etc)?

**Comment by eanewman** [comment]
I think the questions will be constantly evolving since this is really just the first cut, but this is some good material to work with :). This is a good opportunity to collaborate not only with Cindy, who is working on the coding side of the question task, but also with Ryland, who is working on the core logic. It's good to start thinking about how the questions and the data you collect from each question can be used to directly feed into the core algorithm for computing the current score. Again, this is definitely going to be an iterative process! Just something to think about.

**Comment by eliyahubowers** [comment]
Cover up: just means to make it seem like your dealings are not as shady as they actually are.
license removal of forest: give other people a license to cut down trees (for a price).
They are the natives of the forest (the people living there).

**Review by ChaoticLeah — CHANGES_REQUESTED** [review]
for question 1, "Turn wood into goods", if you already have the wood, why would it change people's opinion on you? (I think it needs some rewording) Also if you did something bad in the past and you don't pay to cover up your bad deeds you might want to make it so there is a chance that they find out and hate you a bit more.

**Pull request closed without merging**